docs/querying/basics.md

```diff
@@ -165,12 +194,12 @@ following units:
 * `s` - seconds
 * `m` - minutes
 * `h` - hours
-* `d` - days - assuming a day has always 24h
-* `w` - weeks - assuming a week has always 7d
-* `y` - years - assuming a year has always 365d
+* `d` - days - assuming a day has 24h
+* `w` - weeks - assuming a week has 7d
+* `y` - years - assuming a year has 365d
 
-Time durations can be combined, by concatenation. Units must be ordered from the
-longest to the shortest. A given unit must only appear once in a time duration.
+Time durations can be combined by concatenation. Units must be ordered from the
+longest to the shortest. A given unit must only appear once in the time duration.
```

Comment:
Why changing this?

Reply:
These are small, but here are my thoughts:

It's an independent clause (a self-sustaining thought), which is then followed by a modifier.

The word "the" instead of "a" overloads subject defined in the first sentence.